addons/test/autoCorrectedMmddyyyyPipe.spec.js

```diff
@@ -28,8 +28,4 @@ describe('autoCorrectedMmddyyyyPipe', () => {
   it('returns false if day 1st digit is 0 and 2nd digit is also 0', () => {
     expect(autoCorrectedMmddyyyyPipe('12/00')).to.equal(false)
   })
-
-  it('completes the year to `200` if the 1st digit is 0', () => {
```

Comment:
The only thing I primarily was concerned about is the auto-completion on Month/Day. Removing auto completion on the year is okay with me.